.vscode/settings.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Remove this file from the list of changes

Reply:
updated